src/lib.rs

```diff
@@ -148,11 +148,14 @@
 //! them once. **Omitting them in the following cargo invocations will not undo
 //! their effects!**
 
-#![cfg_attr(feature = "cargo-clippy", allow(let_unit_value))]
-#![cfg_attr(feature = "cargo-clippy", allow(new_without_default))]
-#![cfg_attr(feature = "cargo-clippy", allow(type_complexity))]
-#![cfg_attr(feature = "cargo-clippy", allow(trivially_copy_pass_by_ref))]
-#![cfg_attr(feature = "cargo-clippy", allow(derive_hash_xor_eq))]
+#![deny(clippy::all)]
```

Comment:
IMHO better do this on travis level
```
cargo clippy -- -D warnings
```

Reply:
:+1:

Reply:
My thought here was that people would probably miss it if it were CI only, but if you want I can shift it.  @GuillaumeGomez @sdroege ?

Reply:
If it's CI only, it's fine so it means it won't break in case a new lint is added. I personally think this is the best solution. And we won't miss it since releases have to go through CI.

Reply:
OK, shifted.